frontend/src/assets/css/variables.scss

```diff
@@ -11,4 +11,7 @@
   --pink-200: #e1a7d5;
   --red-300: #d51a52;
   --white-alpha-80: #ffffff80;
+  --toastify-color-error: var(--red-300);
+  --toastify-icon-color-error: var(--red-300);
+  --toastify-color-progress-error: var(--red-300);
```

Comment:
let's not complicate logic with variables and just use them explicitly, without additional variables for each component
```suggestion
```